cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Payment Card Industry Data Security Standard (PCI DSS) compliance in CockroachDB Cloud Dedicated
+summary: Learn about compliance with Payment Card Industry Data Security Standard (PCI DSS) for CockroachDB Cloud Dedicated clusters.
+toc: true
+docs_area: manage.security
+---
+
+The Payment Card Industry Data Security Standard (PCI DSS) is a series of requirements for the safe handling of sensitive data associated with credit and debit cards. In the PCI DSS standard, and in the rest of this article, this data is referred to as "cardholder data". When implemented correctly, PCI DSS helps to protect your organization and your customers from fraud. PCI DSS is mandated by credit card issuers but administered by the [Payment Card Industry Security Standards Council](https://www.pcisecuritystandards.org/).
+
+{% include v22.1/dedicated-pci-compliance.md %}
+
+Responsibility for compliance with PCI DSS is shared among multiple parties, including card issuers, banks, software-as-a-service (SaaS) providers, and retail merchants. Compliance measures are implemented as a series of business practices, security controls, and technological solutions. An organization's compliance with PCI DSS is certified by a PCI Qualified Security Assessor (QSA).
+
+This article provides information about {{ site.data.products.dedicated }}'s compliance with PCI DSS, describes some of the ways that {{ site.data.products.db }} implements and enforces compliance, and illustrates some of the types of changes you may need to implement outside of your {{ site.data.products.dedicated }} clusters. For detailed instructions and working examples, refer to [Integrate {{ site.data.products.dedicated }} with Satori](/docs/{{site.versions["stable"]}}/satori-integration.html).
+
+## Overview of PCI DSS
+
+When a system complies with PCI DSS, the system meets the goals of the standard by implementing a series of twelve requirements, according to a PCI Qualified Security Assessor (QSA). The following table, which is published in Payment Card Industry Security Standards Council's [PCI DSS Quick Reference Guide, version 3.2.1](https://listings.pcisecuritystandards.org/documents/PCI_DSS-QRG-v3_2_1.pdf), summarizes the goals and requirements of PCI DSS.
+
+<table>
+<tgroup cols="2">
+<thead>
+<tr>
+  <th>Goal</th>
+  <th>PCI DSS Requirement</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td><i>Build and Maintain a Secure Network and Systems</i></td>
+  <td><ol>
+          <li>Install and maintain a firewall configuration to protect cardholder data</li>
```

Comment:
All these numbered items are incremented continuously in every row in the table. I don't think that was the intention was it?

Reply:
OK, I see the `ol start` annotation but it seems odd that the numbering continues.

Reply:
It was the intention, as it's exactly how the table is in the source document.